ts/nni_manager/rest_server/nniRestServer.ts

```diff
@@ -19,7 +20,7 @@ import { createRestHandler } from './restHandler';
  */
 @component.Singleton
 export class NNIRestServer extends RestServer {
-    private readonly API_ROOT_URL: string = '/api/v1/nni';
+    private readonly API_ROOT_URL: string = `/api/v1/nni/${getUrlPrefix()}`;
```

Comment:
I think prefix URL should better override `/api/v1/nni`.

Reply:
The same opinion with @liuzhe-lz , in https://github.com/microsoft/nni/blob/5a71ce916b9298d7d3cda77b3fee24d4474f05e8/nni/tools/nnictl/url_utils.py#L10

so that we don't need to modify too many internal interfaces.

Reply:
Changed.